hail/python/hail/genetics/pedigree.py

```diff
@@ -294,10 +298,10 @@ def write(self, path):
 
         .. caution::
 
-            Phenotype information is not preserved in the Pedigree data structure in Hail.
-            Reading and writing a PLINK .fam file will result in loss of this information.
-            Use the key table method :meth:`~hail.KeyTable.import_fam` to manipulate this
-            information.
+            Phenotype information is not preserved in the Pedigree data
+            structure in Hail.  Reading and writing a PLINK .fam file will
+            result in loss of this information.  Use the key table method
```

Comment:
You still say "key table", though you've taken out the dead `KeyTable` link.

Reply:
gone